backend/uclapi/uclapi/urls.py

```diff
@@ -16,8 +16,9 @@
 from django.conf.urls import url, include
 from django.contrib import admin
 from dashboard.views import documentation, home, about
+from oauth.views import settings, logout
 from marketplace.views import marketplace
-
+from django.urls import path
```

Comment:
Please move this line to just under `from django.contrib import admin`

Reply:
Done!